src/snowflake/snowpark/_internal/analyzer/binary_plan_node.py

```diff
@@ -187,3 +194,23 @@ def __init__(
     @property
     def sql(self) -> str:
         return self.join_type.sql
+
+    @property
+    def individual_complexity_stat(self) -> Counter[str]:
```

Comment:
I am renaming this to `individual_node_complexity` and `cumulative_node_complexity`. since the return value is a dictionary, adding `score` could also be misleading.

Reply:
I earlier had stat as a short hand for statistic